### PR TITLE
fixes #156 - Skywallet branding

### LINKED ISSUE #156: Skywallet branding
**Feature description**

Reference SkyWallet .

**Is your feature request related to a problem? Please describe.**
The inherited technology is mentioned all over across the project tree .

**Additional context**
noticed after merging #147 

**Possible implementation**

- [x] rename source files
- [x] update file contents
- [x] ensure output binary files are created with correct names

### PR DESCRIPTION
**Pull request by stdevMac**
Fixes #156 

Changes:
- rename source files
- update file contents
- ensure output binary files are created with correct names

Does this change need to mentioned in CHANGELOG.md?
no

Requires testing
no

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +56,13 @@ Follow the instructions written on [tiny-firware/README.md](https://github.com/s
 
 ### Build and run emulator
 
+Update submodules:
+
+```
+git submodule update --init --recursive
+```
```
Comment (olemis) [inline README.md]: Merge both code blocks into a single one .

Comment (stdevMac) [reply README.md]: Already done here c98848ccdca4f89848be7bae5d670ec70e85f8fe

**Comment by stdevMac** [comment]
How should i name it? @olemis

**Comment by olemis** [comment]
superseded by #168

**Pull request closed without merging**